cdap-docs/_common/_themes/cdap-intro/theme.conf

```diff
@@ -0,0 +1,17 @@
+[theme]
+inherit = cdap
+stylesheet = cdap-intro.css
+pygments_style = tango
+
+[options]
+docs_url = ''
+language = 'en'
+manual = ''
+manuals = []
+manual_titles = []
+manual_icons = []
+meta_git = []
+stickysidebar = true
+release = ''
```

Comment:
Yes, they set the options, and show (document) what type they should be. They are then overwritten in particular uses of the theme. And, if they aren't, they are an empty string, rather than a `None`, which would cause problems when they are added to a string...